packages/core/src/prompts/select-key.ts

```diff
@@ -0,0 +1,26 @@
+import Prompt, { PromptOptions } from './prompt';
+
+interface SelectKeyOptions<T extends { value: any }> extends PromptOptions<SelectKeyPrompt<T>> {
+	options: T[];
+}
+export default class SelectKeyPrompt<T extends { value: any }> extends Prompt {
+	options: T[];
+	cursor: number = 0;
+
+	constructor(opts: SelectKeyOptions<T>) {
+		super(opts, false);
+
+		this.options = opts.options;
+		const keys = opts.options.map(v => v.value[0]);
```

Comment:
If we want to restrict the value type, we could define a more narrow type:

```ts
type LowerCaseChar = 'a' | 'b' | 'c' | ... | 'z'
type Char = LowerCaseChar | UpperCase<LowerCaseChar>
type Digit = 0 | 1 | 2 | ... | 9
type AlphaNumericChar = Char | Digit
```

Of course we wouldn't support all kinds of unicode symbols, but I think it would suffice.

Reply:
Ignore. Irrelevant as hotkeys are generated from initial of each word.